src/lib/features/metrics/instance/instance-service.ts

```diff
@@ -105,6 +111,7 @@ export default class ClientInstanceService {
         value.clientIp = clientIp;
         value.createdBy = SYSTEM_USER.username!;
         this.seenClients[this.clientKey(value)] = value;
+        this.eventBus.emit(CLIENT_REGISTERED, value);
```

Comment:
it's slightly confusing the we have CLIENT_REGISTER and CLIENT_REGISTERED events